hoodie-client/src/test/java/com/uber/hoodie/index/TestHbaseIndex.java

```diff
@@ -35,8 +35,10 @@
 import com.uber.hoodie.config.HoodieIndexConfig;
```

Comment:
Can you create a TestQPSAllocatorClass that is performing some functionality that simulates failures and make sure that it is handled ?

Reply:
any thoughts here to test failure scenario inside HbaseIndex ?

Reply:
yes will add a separate test class. sry, forgot to comment that I am working on it.

Reply:
done, added TestHBaseQPSResourceAllocator class.